public/index.php

```diff
@@ -1,8 +1,8 @@
 <?php
-require __DIR__ . '/../vendor/autoload.php';
-require __DIR__ . '/../config/env.php';
-require __DIR__ . '/../vendor/yiisoft/yii2/Yii.php';
+require dirname(__DIR__) . '/vendor/autoload.php';
```

Comment:
I find `__DIR__ . '/../'` more readable than the usage of `dirname()`.

Reply:
I could also say the same about `dirname(__DIR__)` being more readable to me, why I sent a PR is more about being consistent `dirname(__DIR__)` is being used everywhere other than index.php take a look at `config/web.php` and `config/console.php`

Reply:
I'd prefer to change those `config/web.php` and `config/console.php`

Reply:
Since PHP 7.0 `dirname()` has a second parameter for levels, so you can use `dirname(__DIR__, 2)`.

I'm for creating `APP_ROOT` constant and use it as a root to build paths. DRY.